bot/cogs/convert/__init__.py

```diff
@@ -0,0 +1,11 @@
+import logging
+
+from bot.bot import Friendo
+from ._cog import Convert
+
+logger = logging.getLogger("convert")
```

Comment:
We can remove logging from here as it is not used.

Reply:
I completely agree that removing unused logging code is a good idea. Not only does it make the code more streamlined and easier to read, but it also helps to prevent any unnecessary cluttering of the log files. Additionally, removing unused logging code can help to improve the overall performance and efficiency of the code. I think it's definitely worth taking the time to clean up and remove any unnecessary logging lines.